### PR TITLE
Improvement: publishing scripts

### PR DESCRIPTION
**Pull request by artemii-yanushevskyi**

### CODE REVIEW

File: content/getting-started/yaml.md [hunk 1]
```diff
@@ -297,12 +297,38 @@ publishing:
...
+        if [ -a "~/SUCCESS" ] ; then
+           # build successful
+        else
+           # build failed
+        fi
 ```
 
+Note that in this example the `SUCCESS` file will not be created if some build step failed. We it's only used to distinguish between failed and successfull builds. 
+
+Unlike the scripts run in the scripts section before the publishing step, the post-publish scripts are run regardless of the status of previous scripts.
```
Comment (helinanever) [inline content/getting-started/yaml.md:330]: let's move this right after the intro paragraph for ### Publishing

File: content/getting-started/yaml.md [hunk 2]
```diff
@@ -290,17 +290,40 @@ There are several things to keep in mind about patterns:
...
   email:
     recipients:
       - name@example.com
+  scripts:
+    script: |
+      if ls /path/to/artifacts/**/*.ipa 1> /dev/null 2>&1; then
+        # publish artifacts
+      fi 
+```
+
```
Comment (helinanever) [inline content/getting-started/yaml.md:308]: This requires some explanation here, e.g.:
You can also use the post-publishing scripts to report build status.

**Review by helinanever — CHANGES_REQUESTED** [review]

**Review by helinanever — APPROVED** [review]
LGTM

**Review by mikhail-tokarev — CHANGES_REQUESTED** [review]

File: content/getting-started/yaml.md [hunk 3]
```diff
@@ -297,12 +297,38 @@ publishing:
...
+    - name: Report build status
+      script: |
+        if [ -a "~/SUCCESS" ] ; then
+           # build successful
+        else
+           # build failed
+        fi
 ```
 
+Note that in this example the `SUCCESS` file will not be created if some build step failed. We it's only used to distinguish between failed and successfull builds.
```
Comment (helinanever) [inline content/getting-started/yaml.md]:
```suggestion
Note that in this example the `SUCCESS` file will not be created if some build step failed. It is only used to distinguish between failed and successful builds. 
```

File: content/getting-started/yaml.md (hunk 1 above)
Comment (helinanever) [inline content/getting-started/yaml.md]:
```suggestion
Unlike the scripts run in the scripts section before the publishing step, the publishing scripts are run regardless of the status of previous scripts.
```

**Pull request merged**